### PR TITLE
Makefile "release" target updated with yaml generation

### PR DESCRIPTION
**Pull request by tzununbekov**
@sameersbn please take a look at possible solution for release yaml generation

### CODE REVIEW

**Comment by tzununbekov** [comment]
I'm not sure what should be in Makefile and what in CI pipeline. We can move `kubectl` command back to CircleCI config

File: Makefile [hunk 1]
```diff
@@ -68,6 +71,8 @@ release: ## Build release binaries
 			GOOS=$${GOOS} GOARCH=$${GOARCH} $(GO) build -ldflags "$(LDFLAGS)" -o $(DIST_DIR)/$${RELEASE_BINARY} ./cmd/$$bin ; \
 		done ; \
 	done
+	$(KUBECTL) create -f config --dry-run=client -o yaml |\
+	  $(SED) 's|ko://github.com/triggermesh/bumblebee/cmd/\(.*\)|gcr.io/triggermesh-private/\1:'${IMAGE_TAG}'|' > $(DIST_DIR)/transformation.yaml
```
Comment (sameersbn) [inline Makefile:75]:
```suggestion
	  $(SED) 's|ko://github.com/triggermesh/bumblebee/cmd/\(.*\)|gcr.io/$(IMAGE_REPO)/\1:'${IMAGE_TAG}'|' > $(DIST_DIR)/transformation.yaml
```

Also update `IMAGE_REPO` to be `gcr.io/triggermesh-private`

**Review by sameersbn — APPROVED** [review]
changes lgtm, just a small nitpick

**Comment by sameersbn** [comment]
> I'm not sure what should be in Makefile and what in CI pipeline. We can move kubectl command back to CircleCI config

I think it's fine as part of the `release` target

**Pull request merged**